packages/theming/src/utils/arrowStyles.ts

```diff
@@ -147,7 +147,22 @@ const positionStyles = (position: ARROW_POSITION, size: string, inset: string) =
 export default function arrowStyles(position: ARROW_POSITION, options: ARROW_OPTIONS = {}) {
   const size = options.size || '6px';
   const inset = options.inset || '0';
-  const squareSize = math(`${size} * 2 / sqrt(2)`);
+  const squareSize = math(`${size} * 2 / sqrt(2)`, {
+    symbols: {
+      sqrt: {
+        func: {
+          symbol: 'sqrt',
+          f: (a: number) => Math.sqrt(a),
+          notation: 'func',
+          precedence: 0,
+          rightToLeft: 0,
+          argCount: 1
+        },
+        symbol: 'sqrt',
+        regSymbol: 'sqrt\\b'
+      }
+    }
```

Comment:
Follow https://github.com/styled-components/polished/issues/550 for fix details.